tasks/build/Gaia/COMMIT.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-# This is the commit of the SDK version to use for building Gaia.  We use an
+# This is the SDK release version to use for building Gaia.
 # explicit hash instead of a tag so we don't have to trust GitHub.
-export COMMIT=4c5038018163fbb51ad882c0505df4958dec2ea8
+export COMMIT=v0.33.0
```

Comment:
can we move this someplace more obvious while we are on it? Like on in `config.js`?

Reply:
+ renaming the file from `COMMIT` to `VERSION`

Reply:
the problem is that this is used only in bash / CI, I would prefer to have it either sh or just an file with the string I will put in the TAG variable

Reply:
kk let's create an issue for that ?